sites/firstmedia.com/firstmedia.com.config.js

```diff
@@ -0,0 +1,51 @@
+const dayjs = require("dayjs");
+const timezone = require("dayjs/plugin/timezone");
+const utc = require('dayjs/plugin/utc');
+
+dayjs.extend(timezone);
+dayjs.extend(utc);
+
+module.exports = {
+  site: "firstmedia.com",
+  days: 1,
```

Comment:
```suggestion
  days: 1,
  request: {
    timeout: 30000 // 30 seconds
  },
```

Prevents a timeout error:

```sh
[1/1] firstmedia.com - ABCAustralia.au - Jun 19, 2023 (0 programs)
  Error: Connection timeout
```

https://github.com/freearhey/epg-grabber#site-config

Reply:
hmm, that didn't work for me?
```
Starting...
Loading 'sites/firstmedia.com/firstmedia.com.config.js'...
Loading 'sites/firstmedia.com/firstmedia.com.channels.xml'..
[1/144] firstmedia.com - ABCAustralia.au - Jun 21, 2023 (0 programs)
  Error: connect ETIMEDOUT 2404:5600:0:3::2:443
[2/144] firstmedia.com - AlJazeeraChannel.qa - Jun 21, 2023 (0 programs)
  Error: connect ETIMEDOUT 2404:5600:0:3::2:443
```

Reply:
I get the error all the time without a `timeout`:

```sh
npx epg-grabber --config=sites/firstmedia.com/firstmedia.com.config.js --channels=sites/firstmedia.com/firstmedia.com.channels.xml --output=guide.xml
Starting...
Loading 'sites/firstmedia.com/firstmedia.com.config.js'...
Loading 'sites/firstmedia.com/firstmedia.com.channels.xml'...
[1/144] firstmedia.com - ABCAustralia.au - Jun 21, 2023 (32 programs)
[2/144] firstmedia.com - AlJazeeraChannel.qa - Jun 21, 2023 (31 programs)
[3/144] firstmedia.com - AlQuranAlKareemTV.sa - Jun 21, 2023 (0 programs)
  Error: Connection timeout
```

and with it, everything works:

```sh
npx epg-grabber --config=sites/firstmedia.com/firstmedia.com.config.js --channels=sites/firstmedia.com/firstmedia.com.channels.xml --output=guide.xml
Starting...
Loading 'sites/firstmedia.com/firstmedia.com.config.js'...
Loading 'sites/firstmedia.com/firstmedia.com.channels.xml'...
[1/144] firstmedia.com - ABCAustralia.au - Jun 21, 2023 (32 programs)
[2/144] firstmedia.com - AlJazeeraChannel.qa - Jun 21, 2023 (31 programs)
[3/144] firstmedia.com - ZhejiangSatelliteTV.cn - Jun 21, 2023 (26 programs)
```

Reply:
still didn't work for me but I guess it's not related to the config I guess so alright then